### PR TITLE
CLI commands for parser

### PR DESCRIPTION
**Pull request by tliron**
Also includes some fixed to TOSCA extension.

### CODE REVIEW

File: aria/cli/commands.py [hunk 1]
```diff
@@ -295,3 +294,68 @@ def _load_workflow_handler(handler_path):
         except AttributeError:
             # todo: exception handler
             raise
+
+from ..parser.consumption import (ConsumptionContext, ConsumerChain, Read, Validate, Model, Types, Inputs, Instance)
```
Comment (dankilman) [inline aria/cli/commands.py]: why are the imports here and not at the top?

File: aria/cli/commands.py [hunk 2]
```diff
@@ -164,10 +162,11 @@ def _workspace_setup(self):
         return local_storage()
 
     def _parse_blueprint(self, blueprint_path, inputs=None):
-        plan = parse_from_path(blueprint_path)
-        self.logger.info('blueprint parsed successfully')
-        deployment_plan = prepare_deployment_plan(plan=plan.copy(), inputs=inputs)
-        return plan, deployment_plan
+        #plan = parse_from_path(blueprint_path)
```
Comment (dankilman) [inline aria/cli/commands.py]: can we implement this functionality using the new parser? if not, let's remove the commented lines

File: aria/cli/cli.py [hunk 3]
```diff
@@ -44,6 +46,7 @@ def __init__(self, *args, **kwargs):
         self.commands = {
             'init': InitCommand.with_logger(base_logger=self.logger),
             'execute': ExecuteCommand.with_logger(base_logger=self.logger),
+            'parse': ParseCommand.with_logger(base_logger=self.logger),
```
Comment (dankilman) [inline aria/cli/cli.py]: why did you decide to leave out `spec`?

File: README.md [hunk 4]
```diff
@@ -1,4 +1,193 @@
-Aria
+ARIA
 ====
 
-See http://ariatosca.org/
+[ARIA](http://ariatosca.org/) is a minimal TOSCA orchestrator, as well as a platform for building
+TOSCA-based products. Its features can be accessed via a well-documented Python API, as well as a
```
Comment (dankilman) [inline README.md]: no language-agnostic RESTful API for now

File: README.md [hunk 5]
```diff
@@ -1,4 +1,193 @@
...
+	git clone http://git-wip-us.apache.org/repos/asf/incubator-ariatosca.git
+	cd incubator-ariatosca
+	pip install -e .
+
+To run tests:
+
+	pip install tox
+	tox
+
+To build the documentation:
```
Comment (dankilman) [inline README.md]: we currently have no make file (or docs), so probably best to remove this line

**Comment by dankilman** [comment]
Also, please amend to commit message to include `ARIA-XXXX` where `XXXX` is the JIRA related to this commit

**Comment by ran-z** [comment]
Please see this document regarding commit message guidelines:
https://cwiki.apache.org/confluence/display/ARIATOSCA/Commit+Message+Guidelines

**Pull request merged**